rsmtool/rsmpredict.py

```diff
@@ -88,41 +93,68 @@ def fast_predict(
 
         This dataframe should be read from the "feature.csv" file under the
         "output" directory of the previously run RSMTool experiment.
+
+        Defaults to ``None``.
+    trim : bool
```

Comment:
```suggestion
    trim : bool, optional
```

Reply:
I have changed this. However, in my opinion I think it should be reserved perhaps for parameters that could be ``None`` rather than those that simply have a default. In typing, we use `Optional` to mean some value or `None`. I like to stick to that meaning. But I think the meaning in this codebase is a bit broader.

Reply:
Yeah, that's true. When (If?) we go through the process of adding type hints to everything, we will resolve this inconsistency.